assets/prefabs/Handwhisk.prefab

```diff
@@ -0,0 +1,26 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Cooking:handwhisk",
+        "stackId": "Cooking:handwhisk"
+    },
+    "DisplayName": {
+        "name": "Hand Whisk"
+    },
+    "Durability": {
+        "durability": 300,
```

Comment:
Curiously did you mean to start the item at 75% durability? Usually new items don't start partly damaged :-)